src/View/SSViewer_Scope.php

```diff
@@ -123,16 +124,23 @@ public function __construct($item, SSViewer_Scope $inheritedScope = null)
      * Returns the current "active" item
      *
      * @return object
+     * @deprecated 5.4.0 use getCurrentItem() instead.
      */
     public function getItem()
     {
+        Deprecation::notice('5.4.0', 'use getCurrentItem() instead.');
```

Comment:
Just renamed it to be a little clearer that it's not _just_ getting the value of the `item` property. I ran into problems when I tried to replace all instances of `$this->item` with calling this method, because sometimes that's not what this method returns.